Gemfile.lock

```diff
@@ -24,6 +24,7 @@ PATH
   specs:
     decidim-admin (0.0.1.alpha5)
       active_link_to (~> 1.0.0)
+      carrierwave (~> 1.0.0.beta)
```

Comment:
Personally I prefer `carrierwave` over `paperclip`. If we are using Rails 5 we are gonna deal with a lot of "beta" things so let's stick with it.